source/features/show-pr-file-state.tsx

```diff
@@ -0,0 +1,59 @@
+import select from 'select-dom';
+import oneTime from 'onetime';
+import {observe} from 'selector-observer';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {observeOneMutation} from '../helpers/simplified-element-observer';
+
+function loadJumpList(jumpList: Element): void {
+	jumpList.parentElement!.dispatchEvent(new MouseEvent('mouseover'));
+}
+
+async function init(): Promise<void> {
+	const jumpList = await elementReady('details.toc-select details-menu')!;
+	loadJumpList(jumpList!);
+	// The event listener might not have been attached yet, so we can try twice
+	setTimeout(loadJumpList, 1000, jumpList!);
+	await observeOneMutation(jumpList!);
```

Comment:
Can you wrap this into a function in this file?
```suggestion
	await loadDeferred(jumpList!);
```

`loadJumpList` would become an inline arrow function _inside_ loadDeferred probably